### PR TITLE
Plugin release post June 2020

### PR DESCRIPTION
**Pull request by NiklasMerz**
### Platforms affected
docs

### Motivation and Context

Release post for recent plugin releases

### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

File: www/_posts/2020-06-13-plugins-release.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+We are happy to announce that we have just released an update to some plugins!
+
+* [cordova-plugin-inappbrowser@4.0.0](https://www.npmjs.org/package/cordova-plugin-inappbrowser)
+* [cordova-plugin-splashscreen@5.0.4](https://www.npmjs.org/package/cordova-plugin-splashscreen)
+
+## Release Highlights
+
+### InAppBrowser
+
+### cordova-common
```
Comment (erisu) [inline www/_posts/2020-06-13-plugins-release.md:19]:
```suggestion
### SplashScreen
```

File: www/_posts/2020-06-13-plugins-release.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Cordova-iOS Platform 6.x will have splash screen integrated and no longer needs the plugin. This releas prevents the plugin from being added to the iOS 6.x platform.
```
Comment (erisu) [inline www/_posts/2020-06-13-plugins-release.md:21]:
```suggestion
In the Cordova-iOS 6.x platform, the splash screen feature has been integrated into its core and no longer needs this plugin. This release prevents the plugin from being added to the iOS 6.x platform.
```

**Comment by erisu** [comment]
Might want to confirm what is the correct way `SplashScreen` or `Splashscreen`. I see more search results using `SplashScreen`.

File: www/_posts/2020-06-13-plugins-release.md (hunk 1 above)
Comment (NiklasMerz) [reply www/_posts/2020-06-13-plugins-release.md:19]:
```suggestion
### Splash Screen
```

Looks like Splash Screen is common.

File: www/_posts/2020-06-13-plugins-release.md (hunk 2 above)
Comment (NiklasMerz) [reply www/_posts/2020-06-13-plugins-release.md]: FYI trying to use it with cordova-ios looks like this:

```
Installing "cordova-plugin-splashscreen" for ios
Plugin doesn't support this project's cordova-ios version. cordova-ios: 6.0.0, failed version requirement: <6.0.0
Skipping 'cordova-plugin-splashscreen' for ios
Adding cordova-plugin-splashscreen to package.json
```

**Comment by NiklasMerz** [comment]
Trying to use it with cordova-ios looks like this:

```
Installing "cordova-plugin-splashscreen" for ios
Plugin doesn't support this project's cordova-ios version. cordova-ios: 6.0.0, failed version requirement: <6.0.0
Skipping 'cordova-plugin-splashscreen' for ios
Adding cordova-plugin-splashscreen to package.json
```

**Comment by NiklasMerz** [comment]
The releases are done. Good to post that?

**Pull request merged**